src/WinRT.Runtime/ComWrappersSupport.net5.cs

```diff
@@ -364,9 +364,25 @@ class ComWrappersHelper
             object thisInstance,

```

Comment:
TIL the `ComWrappersHelper` type exists. Weird one.
- Why is this type not static?
- Why does this exist at all? These methods feel like they should just be factory methods on `IObjectReference`:
```csharp
static IObjectReference CreateUnsafe(
    bool isAggregation,
    object thisInstance,
    IntPtr newInstance,
    IntPtr inner,
    Guid iid);
```

Reply:
This type probably should have been static, I think we just missed doing that when we added it.  It is an interesting proposal whether it should have been on IObjectReference, but we had kept it separate as it was meant to a set of helper methods to help deal with WinUI reference tracking setup.  That was why it was put under a helper class.  For now, leaving it as is.